src/android/PurchasesPlugin.java

```diff
@@ -53,6 +50,14 @@ public void onReceived(@NonNull CustomerInfo customerInfo) {
         callbackContext.sendPluginResult(result);
     }
 
+    @PluginAction(thread = ExecutionThread.UI, actionName = "configure", isAutofinish = false)
+    private void configure(String apiKey, @Nullable String appUserID, boolean observerMode,
```

Comment:
Should we also offer a method that does the configure + setting the delegate (as an extra parameter)? Currently I'm just worried about devs updating to this and their delegates just stop working if they don't set the delegate separately. Though we can just warn people on the CHANGELOG I guess...

Reply:
the delegate should continue to work: the `configure` method in typescript will call both this one and `setupDelegateCallback` (it actually also calls another one for promoted purchases as well). You can only access this method through the typescript method, so it never gets called on its own